forgot_password.html

```diff
@@ -10,29 +10,30 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block">
     <link rel="stylesheet" href="styles/style.css">
     <link href="styles/normalize.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"/>
 </head>
-<body id="forgot_password_page">
+<body>
 <div class="container-fluid">
-    <div class="row" id="logo_invoicer_big_forgot_password_page">
-        <img src="img/logo.svg" width="412" height="146"/>
+    <div class="row" id="invoicer_logo_fp_pg">
+        <img src="img/logo.svg" alt="logo" width="412" height="146"/>
     </div>
     <div class="row text-center">
-        <p id="forgot_password_text">Restore password</p>
+        <p id="fp_txt">Restore password</p>
     </div>
     <div class="row text-center">
-        <p id="confirmation_text">Confirmation of changing password was sent to <span
-                id="user_email_forgot_password_page">{user_email}</span></p>
+        <p id="confirm_txt">Confirmation of changing password was sent to
+            <span>{user_email}</span></p>
     </div>
     <div id="input_email">
-        <div class="row justify-content-center">
-            <div class="d-flex flex-wrap flex-column align-items-center col-xxl-4 col-xl-5 col-lg-6 col-md-6 col-sm-10 col-10"
-                 id="background_input_forgot_password_page">
-                <md-outlined-text-field class="col-12" id="input_forgot_password_page"
+        <div class="row justify-content-center animate__animated animate__fadeInLeft">
+            <div class="bg_input_fp_pg d-flex flex-wrap flex-column align-items-center col-xxl-4 col-xl-5 col-lg-6 col-md-6 col-sm-10 col-10"
+            >
+                <md-outlined-text-field class="col-12" id="em_input_fp_pg"
```

Comment:
do we have any chance to replace em_input_fp_pg to email?

Reply:
change